homeassistant/components/devolo_home_control/config_flow.py

```diff
@@ -23,13 +23,13 @@ def __init__(self):
             vol.Required(CONF_USERNAME): str,
             vol.Required(CONF_PASSWORD): str,
         }
+        self._reauth_entry = None
+        self._url = DEFAULT_MYDEVOLO
 
     async def async_step_user(self, user_input=None):
         """Handle a flow initiated by the user."""
         if self.show_advanced_options:
-            self.data_schema[
-                vol.Required(CONF_MYDEVOLO, default=DEFAULT_MYDEVOLO)
-            ] = str
+            self.data_schema[vol.Required(CONF_MYDEVOLO, default=self._url)] = str
```

Comment:
So now the default is whatever is in self._url, is that really wanted?

Reply:
The URL can actually not change. It might change in the user flow and has to stay the same from there on.